xbmc/GUIInfoManager.cpp

```diff
@@ -6136,11 +6136,11 @@ std::string CGUIInfoManager::GetLabel(int info, int contextWindow, std::string *
     }
     break;
   case PLAYER_PLAYSPEED:
-      if(g_application.m_pPlayer->IsPlaying())
+      if(g_application.GetAppPlayer().IsPlaying())
```

Comment:
all those condition checks are rather useless/wrong and can be dropped
- there is no guarantee that player is still playing when entering the block
- if player is not playing the methods return defaults
- unlikely that a skins query those labels if player is not playing

Reply:
done in a separate commit